dtime/sleepwithcontext_test.go

```diff
@@ -57,8 +58,15 @@ func TestSleep(t *testing.T) {
 			start := time.Now()
 			SleepWithContext(ctx, tcinfo.Arg)
 			actual := time.Since(start)
-			assertDurationEq(t, tcinfo.Expected, actual,
-				time.Second/100)
+
+			slop := 10 * time.Millisecond
```

Comment:
Dumb question, does `slop` stand for something or is it just the word that was used to reference for a time buffer of sorts?

Reply:
It's just the word.